Resources/Prototypes/Entities/Structures/Doors/Airlocks/airlocks.yml

```diff
@@ -21,6 +21,8 @@
   components:
   - type: Sprite
     sprite: Structures/Doors/Airlocks/Standard/atmospherics.rsi
+  - type: WiresPanel
+    securityLevel: Level2
```

Comment:
Can't the WiresPanel infer what security level it's at by looking at what node it's on in the construction component? This seems like a way to introduce invalid states. What happens when somebody specifies the wrong node and securityLevel combination?

Reply:
Yup, but the build tests complained that component data was being set runtime